packages/platform-android/src/commands/runAndroid/index.js

```diff
@@ -400,7 +416,7 @@ export default {
       name: '--terminal [string]',
       description:
         'Launches the Metro Bundler in a new window using the specified terminal path.',
-      default: getDefaultUserTerminal,
+      default: getDefaultUserTerminal(),
```

Comment:
How was it working previously?

Reply:
it either didn't or used `open`/`sh` to open with a default program